documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -11,46 +11,329 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
-** _ReferenceElement_ is serialized as $\{ReferenceElement/idShort}: $\{ReferenceElement/value} where $\{ReferenceElement/value} is the serialization of the _Reference_ class.
-** _RelationshipElement_ is serialized as named JSON object with $\{RelationshipElement/idShort} as the name of the containing JSON property. The JSON object contains two JSON properties. The first is named “first”. The second is named “second”. Their corresponding values are $\{RelationshipElement/first} resp. $\{Relationship/second}. The values are serialized according to the serialization of a _ReferenceElement_ (see above).
-** _AnnotatedRelationshipElement_ is serialized according to the serialization of a _ReleationshipElement_ (see above). Additionally, a third named JSON object is introduced with “annotations” as the name of the containing JSON property. The value is $\{AnnotatedRelationshipElement/annotations}. The values of the array items are serialized depending on the type of the annotation data element.
-** _Entity_ is serialized as named JSON object with $\{Entity/idShort} as the name of the containing JSON property. The JSON object contains three JSON properties. The first is named “statements” $\{Entity/statements} and contains an array of the serialized submodel elements according to their respective serialization mentioned in this clause. The second is named either “globalAssetId” or “specificAssetId” and contains either a _Reference_ (see above) or a _SpecificAssetId_. The third property is named “entityType” and contains a string representation of $\{Entity/entityType}.
-** _BasicEventElement_ is serialized as named JSON object with $\{BasicEventElement/idShort} as the name of the containing JSON property. The JSON object contains one JSON property named “observed” with the corresponding value of $\{BasicEventElement/observed} as the standard serialization of the _Reference_ class.
-** _SpecificAssetId_ is serialized as named JSON object with three JSON properties named as the attributes of _SpecificAssetId._
+[[table-value-data-attributes]]
+.Value-Only Attributes
+[%autowidth, width="100%", cols="48%,52%",options="header",]
+|===
+|*Class Name* |*Fields for Value-Only representationfootnote:[ attribute in parenthesis (like e.g. (value)) means that the attribute name is not displayed, only its Value-Only serialization. 
+For the other attributes the attribute itself will be part of the serialization]*
+2+|*Identifiables*
+|AssetAdministrationShell a|assetInformation, submodels
+|Submodel a| (submodelElements)
+
+====
+Qualifiers are ignored. This also holds for qualifiers of kind "ValueQualifier".
+====
+
+|*SubmodelElements* a| 
+
+====
+Qualifiers are ignored. This also holds for qualifiers of kind "ValueQualifier".
+====
+
+|SubmodelElementCollection a| (value) 
+
+====
+If the collection contains submodel elements of type
+ Operation or Capability: these are ignored in the Value-Only serialization
```

Comment:
I am not sure: Operation and capability do not have a Value-Only serialization but this could also mean that if a collection contains them the complete collection is not serialized. But here it is stated that they are ignored and the rest is serialized. I think this needs to be clarified.

Reply:
Indeed, it needs to be defined what is the expected result when a collection is empty or only contains elements that are not serializable. 
My suggestion would be that they could be omitted, i.e. empty elements are optional.

However, duplicating the information that Operation and Capability are not serializable for valueOnly is not really helpful. I suggest changing the test to something similar to
> If a SubmodelElementCollection is empty or contains only SubmodelElements that have no valueOnly serialization, serializing the SubmodelElementCollection may be omitted.

Reply:
I added issue https://github.com/admin-shell-io/aas-specs-api/issues/270 for this discussion.

The discussion with the empty collection seems to be the same as in https://github.com/admin-shell-io/aas-specs/pull/375#discussion_r1558986741

I think a collection cannot be empty in the sense that a value is missing. It is valid that are not SubmodelElements in the collection but I would display it  as an empty object because this is what was defined by the data provider. In the Normal-Serialization "value" could be ommitted but not the collection itself. It would be part of the Value-Only serialization of the Submodel. So it is different to Capability or Operation that are always "empty" w.r.t. this understanding.